### ISSUE TITLE
Problem with _documentation/consumer/dragonboard/dragonboard410c/guides/camera-module.md

### ISSUE THREAD

**Issue by rball325** [96boards/documentation#548]
Please enter some details about the bug/issue so we can promptly resolve it. 
URL:  https://www.96boards.org/documentation/consumer/dragonboard/dragonboard410c/guides/camera-module.md.html

I'm trying to get a camera working with the dragonboard410c, and I've run into a problem with the following section:

"Enable camera
The Linaro release is configured with no camera and users with camera are expected to configure the DTS file accordingly. Please check commit “dts: Disable camera sensors in dtsi” in the kernel. This patch assumes that two OV5645 camera sensors are connected on the board (one per CSI2 channel) and disables them. To enable a camera please revert (part of) this patch."

I cloned the kernel source from http://git.linaro.org/landing-teams/working/qualcomm/kernel.git, but looking at the recent commits I don't see anything like the one mentioned above. 

Am I looking in the right place?

**Comment by daniel-thompson** [comment]
Perhaps we should update the *Enable camera* chapter to point at https://github.com/96boards/dt-update instead of hinting about reverting a commit (without an SHA1).

dt-update isn't perfect but https://github.com/96boards/dt-update/blob/master/scripts/db410c/overlays/db410c-d3-mezzanine.dts should give the user more clues about how to modify that DT than the current text.

**Comment by Mani-Sadhasivam** [comment]
@rball325 As pointed out by @daniel-thompson, please move this topic to 96Boards Forum if you'd like to have a technical discussion.

Thanks for your time!

**Comment by daniel-thompson** [comment]
@Mani-Sadhasivam 
I did **not** advise Rick to move to this topic to the forum. I just wanted to be clear that the discussion on the issue would be about how to improve the docs rather than solve any specific issue.

IMHO we should consider this as a bug report! The doc referenced was and remains extremely vague about how to find the code needed to modify the device tree.

**Comment by Mani-Sadhasivam** [comment]
@daniel-thompson Oops, sorry for that!

> IMHO we should consider this as a bug report! The doc referenced was and remains extremely vague about how to find the code needed to modify the device tree.

Agree with you. Eventhough we haven't settled with a way to modify devicetree at runtime, I would like to see the use of `dt-update` on cases like this...

@ndechesne @todortomov What is your opinion?

**Comment by ndechesne** [comment]
yes, this part needs to be improved, and this is planned for the next release. we are reworking the whole camera setup, for that purpose.

**Comment by Mani-Sadhasivam** [comment]
@ndechesne Thanks for the update. Will keep this issue opened until we have the PR for improving the Camera guide.

**Comment by sdrobertw** [comment]
@Mani-Sadhasivam please issue your changes to the DragonBoard document in question:
https://www.96boards.org/documentation/consumer/dragonboard/dragonboard410c/guides/camera-module.md.html

This issue must be closed prior to SAN19.

**Comment by ole00** [comment]
Hi, is there any update on this? I'm also looking for the commit  “dts: Disable camera sensors in dtsi” in order to find out how to configure camera via CCI , but can not find the commit. Could you please state:
1) which repository (git url)
2) which branch 
3) which commit hash ? 

Thank you.
